web/crux/src/app/team/team.service.ts

```diff
@@ -289,7 +288,13 @@ export default class TeamService {
     let hasSession = false
 
     if (!user) {
-      user = await this.kratos.createUser(request.email)
+      user = await this.kratos.createUser({
+        email: request.email,
+        name: {
+          first: request.firstName,
+          last: request.lastName ?? undefined,
```

Comment:
I am not the expert but you can't change this to `request.?lastName`? I'm just curious.